src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -0,0 +1,133 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
+
+import com.google.common.base.Predicate;
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.elemnt.Wait;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+public class AutoplayVuap {
+
+  private static final String SLOT_SELECTOR_PREFIX = "#%s .";
+
+  private static final String PAUSE_CLASS_NAME = "pause-overlay";
+
+  private static final String STOP_CLASS_NAME = "close-ad";
+
+  private static final String CURRENT_TIME_CLASS_NAME = "current-time";
+
+  private static final String PAUSE_BUTTON_SELECTOR_FORMAT = SLOT_SELECTOR_PREFIX + PAUSE_CLASS_NAME;
+
+  private static final String STOP_BUTTON_SELECTOR_FORMAT = SLOT_SELECTOR_PREFIX + STOP_CLASS_NAME;
+
+  private static final String CURRENT_TIME_SELECTOR_FORMAT = SLOT_SELECTOR_PREFIX + CURRENT_TIME_CLASS_NAME;
```

Comment:
It all makes sense and supports DRY but there is a grepping issue as well: if somebody will do a clean-up and will use `grep` to find if any test is using `#TOP_LEADERBOARD .close-ad` she won't find it, will she? My proposals:
* add comments at the top of `*_FORMAT` statics with the full list of possibilities,
* resign from well-described `*_CLASS_NAME` statics and `SLOT_SELECTOR_PREFIX` and just stay with `*_FORMAT`,
* your ideas.

What do you think about all of it? :)